src/App.js

```diff
@@ -46,6 +52,18 @@ const App = ({ history, location }) => {
 
     }, [appNavClick]);
 
+    useEffect(() => {
+        insights?.chrome?.globalFilterScope?.('vulnerability');
+        insights.chrome.on('GLOBAL_FILTER_UPDATE', ({ data }) => {
```

Comment:
In the line above you are chaining optionally but on this line you don't so in case insights or insights chrome would be nullish, the second line would produce an error.

Reply:
> I pulled lastest insights chrome commit (bb0760e)

Do you run chrome locally? You should be able to run the PR without the chrome locally, just running the proxy. 

> Sometimes the global filter appears on page load and then disappears when systems are loaded (might be some kind of race condition). 

try the above and if it's doesn't solve the issue it would be helpful a reproducer

> Adding filter changes the URL params but doesn't refresh the system list.

Did you run `rm -rf /dist; rm -rf node_modules; npm install`? Cause there is a new javascript-client version.

Reply:
fixed it. 
 it's not the greatest solution but since it's happening only in refresh and it's working fine when you change the page from the side nav or accessing directly the page I won't invest more time unless it's needed it